src/EditorFeatures/CSharpTest2/Recommendations/DelegateKeywordRecommenderTests.cs

```diff
@@ -288,7 +288,7 @@ public async Task TestNotAfterSealed()
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
         public async Task TestAfterStatic()
         {
-            await VerifyAbsenceAsync(SourceCodeKind.Regular, @"static $$");
+            await VerifyKeywordAsync(SourceCodeKind.Regular, @"static $$");
             await VerifyKeywordAsync(SourceCodeKind.Script, @"static $$");
```

Comment:
can we have a test for this after static inside a method as well?

Reply:
Done in b581ce2ac282cc8592047a134758e4d24007955a (with a fix in 8439d03)